src/users/controllers/CreateUserController.js

```diff
@@ -0,0 +1,31 @@
+import { Prisma } from '@prisma/client';
+import { prismaClient } from '../../../database/prismaClient.js';
+import { encryptPassword } from  '../utils/encrypt.js';
+export class CreateUserController {
+    async handle(request, response) {
+        const {name, email, password} = request.body;
+        const hashPassword =  await encryptPassword(password)
+        try {
+            const user = await prismaClient.user.create({
+                data: {
+                    nome: name,
```

Comment:
Why don't name "nome" as name? Keeping your codebase and database in the same language is a important thing

Reply:
I made the change yesterday in the schema and I didn't change it in the controller.
I will upload a commit correcting